### PR TITLE
Add unit 7 lesson 7 return a list

### PR DESCRIPTION
**Pull request by rosiecakes**
Added solution, test, readme, main
Please review
🍔

### CODE REVIEW

File: unit-7-functions-intro/lesson-7-return-a-list/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Return a list
+
+Write a function that takes in a number, then returns a list of even numbers that are less than the input number.
```
Comment (neo1691) [inline unit-7-functions-intro/lesson-7-return-a-list/README.md]: An alternate version of the problem statement:

Write a function that takes in a number N, and returns a list of even numbers Z such as  0 < Z < N. 

Or you can include both the statements. Some people understand better as simple english and some understand mathematical statements easier. :)

File: unit-7-functions-intro/lesson-7-return-a-list/solutions/solution.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+def return_list(num):
```
Comment (neo1691) [inline unit-7-functions-intro/lesson-7-return-a-list/solutions/solution.py]: Here is a one liner solution:

```
def evens(n):
    return [ x for x in range(1, n+1) if x % 2 == 0 and x != n]
```

Comment (neo1691) [inline unit-7-functions-intro/lesson-7-return-a-list/solutions/solution.py]: Just realized you can use range(1, n) and skip the second if condition

File: unit-7-functions-intro/lesson-7-return-a-list/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Return a list
+
+Write a function that takes in a number, then returns a list of even numbers that are less than the input number. 
+
+Do not include 0, nor the input number. 
+
+```python
+return_list(10) == [2, 4, 6, 8]
+return_list(5) == [2, 4]
```
Comment (santiagobasulto) [inline unit-7-functions-intro/lesson-7-return-a-list/README.md]: I really like this assignment Rosie. Good idea.

File: unit-7-functions-intro/lesson-7-return-a-list/solutions/solution.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+    evens = []
+    [evens.append(num) for num in range(1, num) if num % 2 == 0]
```
Comment (santiagobasulto) [inline unit-7-functions-intro/lesson-7-return-a-list/solutions/solution.py]: Don't do `.append()`. Just return the list comprehension.

File: unit-7-functions-intro/lesson-7-return-a-list/README.md (hunk 1 above)
Comment (rosiecakes) [inline unit-7-functions-intro/lesson-7-return-a-list/README.md]: Great idea, I think I'll incorporate both.

File: unit-7-functions-intro/lesson-7-return-a-list/solutions/solution.py (hunk 4 above)
Comment (rosiecakes) [inline unit-7-functions-intro/lesson-7-return-a-list/solutions/solution.py]: Okie dokie, changed to return comp only

**Pull request merged**